src/clusterfuzz/_internal/google_cloud_utils/batch.py

```diff
@@ -31,9 +31,8 @@
 
 _local = threading.local()
 
-MAX_DURATION = '3600s'
-RETRY_COUNT = 1
-TASK_COUNT = 1
+MAX_DURATION = f'{int(60 * 60 * 2.5)}s'
+RETRY_COUNT = 0
```

Comment:
Any reason for removing the retry?

Reply:
It doesn't work at all. It tries to restart a docker container (ClusterFuzz's) that cannot actually be restarted.